test/collection_test.js

```diff
@@ -1456,8 +1456,11 @@ describe("Collection", () => {
     });
   });
 
-  /** @test {Collection#loadDump} */
-  describe("#loadDump", () => {
+  /** 
+   * @deprecated
+   * @test {Collection#loadDump} 
+   */
+  describe("Deprecated #loadDump", () => {
```

Comment:
Same comment about how thoroughly we need/don't need to test `loadDump`.

Reply:
Same question as above.  Should I use something like [chai-spies](https://www.chaijs.com/plugins/chai-spies/)?